sapientml_preprocess/generator.py

```diff
@@ -229,15 +230,43 @@ def generate_code(self, dataset: Dataset, task: Task) -> Tuple[Dataset, Code]:
             logger.warning(
                 f"Symbols that inhibit training and visualization will be removed from column name {str(cols_has_symbols)}."
             )
+            org_df_column = df.columns.values
+            org_target_columns = task.target_columns
+            no_symbol_columns = [col for col in df.columns.values if col not in cols_has_symbols]
             df = df.rename(columns=lambda col: remove_symbols(col) if col in cols_has_symbols else col)
             task.target_columns = [
                 remove_symbols(col) if col in cols_has_symbols else col for col in task.target_columns
             ]
+            rename_dict = {}
+            if df.columns.duplicated().any():
+                same_column = {
+                    k: v
+                    for k, v in collections.Counter(list(df.columns.values)).items()
+                    if v > 1 and k in no_symbol_columns
+                }
+                for target, org_column in zip(df.columns.to_list(), org_df_column.tolist()):
+                    if target in same_column.keys():
+                        rename_dict[org_column] = target + str(same_column[target] - 1)
+                        same_column[target] = same_column[target] - 1
+                    else:
+                        rename_dict[org_column] = target
+
```

Comment:
The current method fails when the renamed names are the same as original names.
For example, there are original columns `Age` , `Age{}` and `Age1`.
In the case, `Age` -> `Age1`, `Age{}` -> `Age0`, so there are two `Age1` columns.
Could you consider the case?